fission/src/ui/modals/configuring/SettingsModal.tsx

```diff
@@ -83,159 +88,189 @@ const SettingsModal: React.FC<ModalPropsImpl> = ({ modalId }) => {
-                    />
-                    <Checkbox
-                        label="Show Score Zones"
-                        defaultState={PreferencesSystem.getGlobalPreference<boolean>("RenderScoringZones")}
-                        onClick={checked => {
-                            setRenderScoringZones(checked)
-                        }}
-                        tooltipText="If disabled, scoring zones will not be visible but will continue to function the same."
-                    />
-                    <Checkbox
-                        label="Show Scene Tags"
-                        defaultState={PreferencesSystem.getGlobalPreference<boolean>("RenderSceneTags")}
-                        onClick={checked => {
-                            setRenderSceneTags(checked)
-                            if (!checked) new SceneOverlayEvent(SceneOverlayEventKey.DISABLE)
-                            else new SceneOverlayEvent(SceneOverlayEventKey.ENABLE)
-                        }}
-                        tooltipText="Name tags above robot."
-                    />
-                    <Checkbox
-                        label="Show Scoreboard"
-                        defaultState={PreferencesSystem.getGlobalPreference<boolean>("RenderScoreboard")}
-                        onClick={checked => {
-                            setRenderScoreboard(checked)
-                        }}
-                    />
-                    <Checkbox
-                        label="Show View Cube"
-                        defaultState={PreferencesSystem.getGlobalPreference<boolean>("ShowViewCube")}
-                        onClick={checked => {
-                            setShowViewCube(checked)
-                        }}
-                        tooltipText="Show the view cube in the top-right corner for quick camera orientation changes."
-                    />
-                    <Checkbox
-                        label="Mute All Sound"
-                        defaultState={PreferencesSystem.getGlobalPreference<boolean>("MuteAllSound")}
-                        onClick={checked => {
-                            setMuteAllSound(checked)
-                        }}
-                    />
-                    <Slider
-                        min={0}
-                        max={100}
-                        value={sfxVolume}
-                        label={"SFX Volume"}
-                        format={{ maximumFractionDigits: 2 }}
-                        onChange={(_, value: number | number[]) => setSFXVolume(value as number)}
-                        tooltipText="Volume of sound effects (%)."
-                    />
-                    {Spacer(8)}
-                </Box>
+                <Tabs
+                    value={activeTab}
+                    onChange={(_, v) => setActiveTab(v as any)}
```

Comment:
You should cast to the concrete type instead
```suggestion
                    onChange={(_, v) => setActiveTab(v as React.SetStateAction<"general" | "graphics">)}
```